com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl

```diff
@@ -38,6 +38,10 @@ struct InputData
 half4 _GlossyEnvironmentColor;
 half4 _SubtractiveShadowColor;
 
+half4 _GlossyEnvironmentCubeMap_HDR;
+TEXTURECUBE(_GlossyEnvironmentCubeMap);
```

Comment:
Not sure if/how we want to re-arrange this, but the texture and sampler are not part of the constant buffer (this is under the header "Constant Buffers")

Reply:
Any suggestions on where it should go? Above the comment? I do think it should be in this file.  Also it would be preferred to keep the `_GlossyEnvironmentCubeMap_HDR` close to the texture.

Reply:
@phi-lira Constant buffers can only contain values, no textures nor samplers. So they're not in any constant buffer, which is why I think it's weird to have under a "Constant Buffers" heading.

Reply:
For the constants buffer are you ok with leaving as is? To me this feels like the right place. There is other stuff under there that isnt in the constant buffer either (the structured buffer for lightdata as an example). I can change the name of the comment if you have any suggestions.